src/Encoder.cpp

```diff
@@ -0,0 +1,193 @@
+#include "Encoder.hpp"
+
+#include <napi.h>
+
+#include "Packet.hpp"
+
+namespace nbs {
+
+    Napi::Object Encoder::Init(Napi::Env& env, Napi::Object& exports) {
+
+        Napi::Function func =
+            DefineClass(env,
+                        "Encoder",
+                        {
+                            InstanceMethod<&Encoder::Write>(
+                                "write",
+                                static_cast<napi_property_attributes>(napi_writable | napi_configurable)),
```

Comment:
See if this works instead
```suggestion
                               napi_property_attributes(napi_writable | napi_configurable)),
```

Reply:
Yep, it works